Source/Core/Core/Analytics.cpp

```diff
@@ -132,17 +132,16 @@ void DolphinAnalytics::ReportGameStart()
 }
 
 // Keep in sync with enum class GameQuirk definition.
-constexpr std::array<const char*, 9> GAME_QUIRKS_NAMES{
-    "icache-matters",
-    "directly-reads-wiimote-input",
-    "uses-DVDLowStopLaser",
-    "uses-DVDLowOffset",
-    "uses-DVDLowReadDiskBca",
-    "uses-DVDLowRequestDiscStatus",
-    "uses-DVDLowRequestRetryNumber",
-    "uses-DVDLowSerMeasControl",
-    "uses-different-partition-command",
-};
+constexpr std::array<const char*, 10> GAME_QUIRKS_NAMES{"icache-matters",
+                                                        "directly-reads-wiimote-input",
+                                                        "uses-DVDLowStopLaser",
+                                                        "uses-DVDLowOffset",
+                                                        "uses-DVDLowReadDiskBca",
+                                                        "uses-DVDLowRequestDiscStatus",
+                                                        "uses-DVDLowRequestRetryNumber",
+                                                        "uses-DVDLowSerMeasControl",
+                                                        "uses-different-partition-command",
+                                                        "uses-di-interrupt-command"};
```

Comment:
I would suggest putting a trailing comma after the last element, so that clang-format doesn't do this wonky formatting.

Technically, given we're on C++17, which has deduction guides, the type specifier can likely also be simplified down to:

```cpp
constexpr std::array GAME_QUIRKS_NAMES{
  // ...
};
```

Reply:
Ahh, that's why it did it that.  I remember that happening before, actually.  I've also simplified the type specification like that (and I can confirm that the static_assert after will give a compile error if it's not updated, which wouldn't be the case with e.g. `std::array<const char*, static_cast<size_t>(GameQuirk::COUNT)>` since it doesn't care if the array is larger than what's actually specified)

Reply:
... and it seems like [OSX](https://buildbot.dolphin-emu.org/#/builders/3/builds/234) and [android](https://buildbot.dolphin-emu.org/#/builders/15/builds/234) hate that, so back to `std::array<const char*, 10>`.

Reply:
Yeah, unfortunately those builders use a version of libc++ that's too old. :/